examples/plotting/file/linked_brushing.py

```diff
@@ -1,3 +1,13 @@
+'''An example of linked brushing with two scatter plots showing two views of the same data.
+   One representing # Cylinders vs. MPG and the second graph representing acceleration vs. MPG
+
+.. bokeh-example-metadata::
+    :sampledata: autompg
+    :apis: bokeh.plotting.Figure.circle
+    :refs: :ref:`plotting_scatter` > :ref:`plotting_scatter_circle`
+    :keywords: scatter, acceleration, mpg, cylinder
```

Comment:
```suggestion
    :keywords: scatter, linked brushing
```
We probably need to provide more clear guidance on what "keywords" is for, but generally, it is to add keywords for specific techniques or plot types